verifier/src/evidence.rs

```diff
@@ -173,13 +173,13 @@ mod test {
         let quote_key = quote_signing_key(quote);
         let tcb_key = tcb_signing_key();
         let mr_signer = [
-            0xD4, 0x12, 0xA4, 0xF0, 0x7E, 0xF8, 0x38, 0x92, 0xA5, 0x91, 0x5F, 0xB2, 0xAB, 0x58,
-            0x4B, 0xE3, 0x1E, 0x18, 0x6E, 0x5A, 0x4F, 0x95, 0xAB, 0x5F, 0x69, 0x50, 0xFD, 0x4E,
-            0xB8, 0x69, 0x4D, 0x7B,
+            0x9F, 0x06, 0xDF, 0x5C, 0xA7, 0x9A, 0x23, 0xFF, 0xDF, 0xB6, 0xCA, 0x0E, 0xC8, 0x55,
```

Comment:
This is due to the updated hw quote. Had to update it to line up with the canned qe_identity.json file.